pkg/multicluster/access/kubeconfig_secret_test.go

```diff
@@ -0,0 +1,66 @@
+package access_test
+
+import (
+	"fmt"
+
+	. "github.com/solo-io/skv2/pkg/multicluster/access"
+	"k8s.io/client-go/tools/clientcmd/api"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var _ = Describe("KubeConfig Secret Conversions", func() {
+	var (
+		clusterName      = "test-name1"
+		kubeConfigFormat = `apiVersion: v1
+clusters:
+- cluster:
+    server: test-server
+  name: %s
+contexts:
+- context:
+    cluster: test-name
+    user: test-name
+  name: test-name
+current-context: test-name
+kind: Config
+preferences: {}
+users:
+- name: test-name
+  user:
+    token: alphanumericgarbage
```

Comment:
lol

Reply:
pulled from service mesh hub, feels like a bit of @GrahamGoudeau test data to me : )